### PR TITLE
Design system updates

### PR DESCRIPTION
**Pull request by noahwc**
## Description of the change

Update visual-dev to reflect updates to the design system. A number of these updates are also specifically related to/required for the Windows, Data, & Programs project.

Full change list:
  - \<Alert>
  - \<Badge>
  - \<Button>
  - \<Card>
  - \<Checkbox>
  - \<DataTable>
  - \<Pagination>
  - \<Row>
  - \<Dropdown>
  - \<Graphics>
  - \<Icon>
  - \<Input>
  - \<Popover>
  - \<Dropdown>
  - \<Form>
  - \<Input>
  - \<Modal>
  - \<RadioAction>
  - \<Select>
  - \<Select2>
  - \<Switch>
  - \<Tag>
  - \<TextArea>
  - \<Tooltip>
 
## Type of change

- [X] Bug fix (non-breaking change that fixes an issue)
- [X] New feature (non-breaking change that adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

- Storybook: https://visual-dev.stencilbook.saasquat.ch/pr294/index.html

### Development

- [x] [Prettier](https://www.npmjs.com/package/prettier) was run (if applicable)
- [ ] The behaviour changes in the pull request are covered by specs
- [ ] All tests related to the changed code pass in development

### Paperwork

- [X] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has a Jira number
- [ ] This pull request has a Process.st launch checklist

### Code review

- [ ] Changes have been reviewed by at least one other engineer
- [X] Security impacts of this change have been considered

### CODE REVIEW

File: packages/visual-dev/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,92 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.3] - 2022-09-
```
Comment (DerekSiemens) [inline packages/visual-dev/CHANGELOG.md:10]: Flagging need for date update

File: packages/visual-dev/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@saasquatch/visual-dev",
-  "version": "1.0.2",
+  "version": "1.0.3-26",
```
Comment (DerekSiemens) [inline packages/visual-dev/package.json:3]: Flagging need for version update before deploy

File: packages/visual-dev/CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,92 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+  - \<Select>
+
+    - `empty` - added empty boolean
+    - `emptySlot` - empty slot to show custom empty views
+    - `emptySlot` - loading slot to show custom loading views
+    - `customIcon` - allow custom icons other than the chevron
+
+### Changed
+
+- Components changed
```
Comment (DerekSiemens) [inline packages/visual-dev/CHANGELOG.md:63]: How were these components changed?

File: packages/visual-dev/CHANGELOG.md (hunk 1 above)
Comment (DerekSiemens) [inline packages/visual-dev/CHANGELOG.md:10]: This feels like a bit more than a patch update, maybe it needs a minor version bump instead?

**Review by zachharrison — APPROVED** [review]

**Pull request merged**